hw-3-new.js

```diff
@@ -0,0 +1,61 @@
+//Task 1
+function durationBetweenDates(date1='31 Jan 2022', date2='1 Jan 2022', word='seconds') {
+    const d1 = new Date(date1);
+    const d2 = new Date(date2);
+    if (word == 'seconds'){
+        return Math.abs((d2.getTime() - d1.getTime())/1000) + ' ' + word;
+    }
+    if (word == 'days'){
+        return Math.abs(d2.getDate() - d1.getDate()) + ' ' + word;
+    }
+    if (word == 'hours'){
+        return Math.abs((d2.getTime() - d1.getTime())/3600000) + ' ' + word;
+    }
+    if (word == 'minutes'){
+        return Math.abs((d2.getTime() - d1.getTime())/60000) + ' ' + word;
+    }
```

Comment:
As you can see there is a lot of copy/paste. Let's try to create a new object with keys - units (it's not a `word`, this is a unit or dimension) and value - numbers. It's can help to fix copy/paste.